src/components/device/ace.ts

```diff
@@ -202,7 +203,6 @@ export default class Ace implements IDeviceManager, IUVCControls {
       this.grpcClient.eraseLogFile(logFile, (err, status: huddly.DeviceStatus) => {
         if (err != undefined) {
           reject(err);
```

Comment:
I think if you remove the return you risk calling reject and then resolve sequentially!